roles/StackStorm.mongodb/vars/debian_20.yml

```diff
@@ -0,0 +1,7 @@
+# Use the following URL to find the key: https://www.mongodb.org/static/pgp/server-{{ mongodb_major_minor_version }}.asc
+mongodb_apt_keys:
+  "4.2": "3F75C80EE56C41F4E49515B62ACD4032058F8B6B"
+  "4.4": "20691EEC35216C63CAF66CE1656408E390CFB1F5"
```

Comment:
Duplicating these keys is not ideal. Is the idea here to prevent people from trying to install 3.6 or 4.0 on debian 20?

Reply:
I don't think that it would prevent people from installing an older version. There is no package for mongodb 3.6 or 4.0 on focal. So trying to install mongodb 4.0 for example would fail while adding the key as such would instead work. 

So the keys aren't needed for U20 and I thought duplication is acceptable as it keeps the code clean and the task to retrieve the variables does not need to be changed: 
```
---
- name: Retrieve variables
  include_vars: "{{ item }}"
  with_first_found:
    - "{{ ansible_os_family | lower }}_{{ ansible_distribution_major_version }}.yml"
    - "{{ ansible_os_family | lower }}.yml"
  tags: [databases, mongodb]
``` 

I think I'm going to add another task that loads the debian.yml that has all the apt keys and the debian_20.yml only if it's a U20 setup to override the mongodb version. Will push it after a test.

Reply:
Done with https://github.com/StackStorm/ansible-st2/pull/298/commits/df3e85cedcca5a63f94dddcfac8cc28b89d0652d